lib/node/runner/checker.go

```diff
@@ -393,8 +398,16 @@ func getMissingTransaction(checker *BallotChecker) (err error) {
 		receivedTransaction = append(receivedTransaction, tx)
 	}
 
+	var bs *storage.LevelDBBackend
+	bs, err = checker.NodeRunner.Storage().OpenBatch()
 	for _, tx := range receivedTransaction {
-		checker.NodeRunner.TransactionPool.Add(tx)
```

Comment:
@spikeekips Why is not `receivedTransaction` saved to `TransactionPool` but only to `SaveTransactionPool`?
In `PushIntoTransactionPool func` a transaction is saved using both.

Reply:
@soonkuk Good point, thanks. The missing tx is only for the proposed ballot of other node, and the tx in `TransactionPool` is used for the proposed ballot of it's node. The tx in `TransactionPool` can be managed separately, I think.